Documentation/platforms/arm/nrf52/boards/nrf9160-dk-nrf52/index.rst

```diff
@@ -2,6 +2,8 @@
 Nordic nRF9160 DK (nRF52840)
 ============================
 
+.. tags:: chip:nrf52, chip:nrf52840, chip:nrf9160
```

Comment:
+ chip:nrf91 ? :-)

According to [1] they recommend nRF9151 for new designs so nRF91 becomes the family too :-)

[1] https://www.nordicsemi.com/Products/nRF9160